interface/controllers/monster_controller.go

```diff
@@ -0,0 +1,27 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/Le-MaliX/ACADEMY-GO-Q42021/services"
+	"github.com/gin-gonic/gin"
+)
+
+func GetAllMonsters(c *gin.Context) {
+	monsters, err := services.GetAllMonsters()
+	if err != nil {
+		c.JSON(http.StatusBadRequest, c.Error(err))
```

Comment:
Refer to the comment I added on the service file to see how to make better error responses